.github/workflows/buildx_dockerhub.yml

```diff
@@ -0,0 +1,71 @@
+# This is a basic workflow to help you get started with Actions
+
+name: buildx for dockerhub
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+    
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+env:
+  # TODO: Change variable to your image's name.
+  IMAGE_NAME: shtripok/pictshare
```

Comment:
wrong image?

Reply:
Your choice.

> Need little customistation:

    set userid and container name on dockerhub (hascheksolutions/pictshare
    for your current)

Reply:
I am not sure what name you want to use. Maybe you prefer to use for the first try other registry or repository name.